shell/platform/android/android_choreographer.cc

```diff
@@ -58,4 +130,40 @@ void AndroidChoreographer::PostFrameCallback(OnFrameCallback callback,
   AChoreographer_postFrameCallback(choreographer, callback, data);
 }
 
+void AndroidChoreographer::PostFrameCallback33(OnFrameCallback33 callback,
+                                               void* data) {
+  AChoreographer* choreographer = AChoreographer_getInstance();
+  AChoreographer_postVsyncCallback(choreographer, callback, data);
+}
+
+int64_t AndroidChoreographer::GetFrameTimelineIndex(
+    AChoreographerFrameCallbackData* data,
+    size_t latency) {
```

Comment:
what are the units of latency and why is it a size_t?

Reply:
Number of frames